.gitignore

```diff
@@ -0,0 +1,14 @@
+# Ignore index
+**/index.md
+
+# Ignore enterprise terms
```

Comment:
It may make sense to remove the below files at the same time.

Reply:
Thanks, I've now done so